src/parallel/mod.rs

```diff
@@ -0,0 +1,122 @@
+//! Parallelization features for ndarray.
+//!
+//! Parallelization features are based on the crate [rayon] and its parallel
+//! iterators. Ndarray implements the parallel iterable traits for arrays
+//! and array views, for some of its iterators and for [Zip].
+//! There are also directly parallelized methods on arrays and on [Zip].
+//!
+//! This requires the crate feature `rayon` to be enabled.
+//!
+//! The following types implement parallel iterators, accessed using these
+//! methods:
+//!
+//! - [Array], [ArcArray]: `.par_iter()` and `.par_iter_mut()`
```

Comment:
These links render fine on nightly, should work in docs.rs as well. I will seem lazy here, I'm hoping we can use these short links. Of course due to bugginess backticks would break the links in this instance.

The rayon link is especially magical.

Reply:
Oh, I didn't realize that feature was already in nightly. That's really convenient!

The automatic linking still works for me when I add backticks (using the 2018-12-02 nightly). Maybe this has been fixed recently and you're using an old nightly?

Reply:
Yes that was it